operator/cmd/actions/start.go

```diff
@@ -26,19 +27,7 @@ var StartCommand = &cli.Command{
 	Action:      operatorMain,
 }
 
-func operatorMain(ctx *cli.Context) error {
-	operatorConfigFilePath := ctx.String("config")
-	operatorConfig := config.NewOperatorConfig(operatorConfigFilePath)
-	err := sdkutils.ReadYamlConfig(operatorConfigFilePath, &operatorConfig)
-	if err != nil {
-		return err
-	}
-
-	operator, err := operator.NewOperatorFromConfig(*operatorConfig)
-	if err != nil {
-		return err
-	}
-
+func updateTelemetryService(operator *operator.Operator, ctx *cli.Context, operatorConfig *config.OperatorConfig) error {
```

Comment:
```suggestion
func sendTelemetryData(operator *operator.Operator, ctx *cli.Context, operatorConfig *config.OperatorConfig) error {
```

Reply:
This could be an operator method instead of receive an operator

Reply:
+1